### PR TITLE
Hover & Pressed color updates

### PR DESCRIPTION
**Pull request by Akatori-Design**
**/ Light Theme**
Added 2 new colors:
- background.hover (#000 2%)
- background.pressed (#000 8%)

Updated 4 colors:
- background.default.hover (grey030 --> #FAFAFA)
- background.default.pressed (grey030 --> #EBEBEB)
- background.alternative.hover (grey100 --> #EDEFF1)
- background.alternative.pressed (grey100 --> #DFE0E2)
image

**/ Dark Theme**
Added 2 new colors:
- background.hover (#FFF 2%)
- background.pressed (#FFF 8%)

Updated 4 colors:

- background.default.hover (grey700 --> #282B2E)
- background.default.pressed (grey700 --> #36383B)
- background.alternative.hover (grey750 --> #191B1D)
- background.alternative.pressed (grey750 --> #27292A)

image

### CODE REVIEW

**Review by brianacnguyen — DISMISSED** [review]

File: src/figma/tokens.json [hunk 1]
```diff
@@ -828,29 +828,39 @@
...
-          "description": "(grey100: #D6D9DC) For component pressed states that use background/alternative",
-          "type": "color"
+          "value": "#DFE0E2",
+          "type": "color",
+          "description": "(#DFE0E2) For component pressed states that use background/alternative"
+        },
+        "hover": {
+          "value": "#00000005",
+          "type": "color",
+          "description": "For component hover states with no background color"
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:858]: Add hex value so it shows up on hover of the token in Figma

```suggestion
          "description": "(#00000005) For component hover states with no background color"
```

File: src/figma/tokens.json [hunk 2]
```diff
@@ -828,29 +828,39 @@
...
+        },
+        "pressed": {
+          "value": "#00000014",
+          "type": "color",
+          "description": "For component pressed states with no background color"
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:863]: Same here
```suggestion
          "description": "(#00000014) For component pressed states with no background color"
```

File: src/figma/tokens.json [hunk 3]
```diff
@@ -1213,29 +1223,39 @@
...
-          "description": "(grey750: #2E3339) For component pressed states that use background/alternative",
-          "type": "color"
+          "value": "#27292A",
+          "type": "color",
+          "description": "(#27292A) For component pressed states that use background/alternative"
+        },
+        "hover": {
+          "value": "#FFFFFF05",
+          "type": "color",
+          "description": "For component hover states with no background color"
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:1253]: Same here
```suggestion
          "description": "(#FFFFFF05) For component hover states with no background color"
```

File: src/figma/tokens.json (hunk 2 above)
Comment (georgewrmarshall) [inline src/figma/tokens.json:863]: Same here
```suggestion
          "description": "(#00000014) For component pressed states with no background color"
```

File: src/figma/tokens.json (hunk 1 above)
Comment (georgewrmarshall) [inline src/figma/tokens.json:858]: Add hex value so it shows up on hover of the token in Figma

```suggestion
          "description": "(#00000005) For component hover states with no background color"
```

File: src/figma/tokens.json [hunk 4]
```diff
@@ -1213,29 +1223,39 @@
...
+        },
+        "pressed": {
+          "value": "#FFFFFF14",
+          "type": "color",
+          "description": "For component pressed states with no background color"
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:1258]: Same here
```suggestion
          "description": "(#FFFFFF14) For component pressed states with no background color"
```

**Review by georgewrmarshall — COMMENTED** [review]
Nice work! Tiny update. Otherwise LGTM!
- searched each color value and made sure they aligned in figma, css, js
- checked storybook Theme Colors stories

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]
Blocking till descriptions have been update

**Review by georgewrmarshall — APPROVED** [review]
🚀

**Comment by Akatori-Design** [comment]
sweet

**Pull request merged**